bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/dialogs/FilteredTree.java

```diff
@@ -176,11 +181,22 @@ public class FilteredTree extends Composite {
 	public FilteredTree(Composite parent, boolean useNewLook, boolean useFastHashLookup) {
 		super(parent, SWT.NONE);
 		this.parent = parent;
+		this.refreshJobDelay = DEFAULT_REFRESH_TIME;
 		if (treeViewer != null) {
 			treeViewer.setUseHashlookup(useFastHashLookup);
 		}
 	}
 
+	/**
+	 * Calls
+	 * {@link #FilteredTree(Composite, int, PatternFilter, boolean, boolean, long)}
+	 * with a default refresh time
+	 */
```

Comment:
```suggestion
	/**
	 * Calls
	 * {@link #FilteredTree(Composite, int, PatternFilter, boolean, boolean, long)}
	 * with a default refresh time.
	 *
	 * @since 3.116
	 */
```